chipsec/modules/tools/cpu/sinkhole.py

```diff
@@ -70,28 +74,28 @@ def check_LAPIC_SMRR_overlap( self ):
         self.logger.log( "[*] Local APIC Base: 0x{:016X}".format(apic_base) )
         self.logger.log( "[*] SMRR Base      : 0x{:016X}".format(smrr_base) )
 
-        self.logger.log( "[*] Attempting to overlap Local APIC page with SMRR region" )
-        self.logger.log( "    writing 0x{:X} to IA32_APIC_BASE[APICBase]..".format(smrrbase) )
+        self.logger.log("[*] Attempting to overlap Local APIC page with SMRR region")
+        self.logger.log( "    Writing 0x{:X} to IA32_APIC_BASE[APICBase]..".format(smrrbase) )
         self.logger.log_important( "NOTE: The system may hang or process may crash when running this test. In that case, the mitigation to this issue is likely working but we may not be handling the exception generated.")
         try:
             self.cs.write_register_field( 'IA32_APIC_BASE', 'APICBase', smrrbase, preserve_field_position=False, cpu_thread=0 )
             ex = False
-            self.logger.log_bad( "Could modify IA32_APIC_BASE to overlap SMRR" )
-        except chipsec.helper.oshelper.HWAccessViolationError:
+            self.logger.log_bad("Could modify IA32_APIC_BASE to overlap SMRR")
```

Comment:
Suggest moving this down to else case (line 95).  I am seeing on a recent platform that an error is not thrown, however the address cannot be changed.

Reply:
An error most likely IS thrown but caught further upstream due to #1177